src/Raven.Server/Json/BlittableJsonDocumentBuilder.cs

```diff
@@ -30,13 +31,14 @@ public enum UsageMode
             ToDisk = ValidateDouble | CompressStrings |  CompressSmallStrings
         }
 
-        internal BlittableJsonDocumentBuilder(RavenOperationContext context, UsageMode mode, string documentId, IJsonParser reader, JsonParserState state)
+        internal BlittableJsonDocumentBuilder(RavenOperationContext context, UsageMode mode, string debugTag, IJsonParser reader, JsonParserState state, int depth = 0)
```

Comment:
@ayende - this was public instead of internal .. somehow the conflict slipped away